src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -1803,13 +1803,15 @@ export default Vue.extend({
             event.preventDefault()
             this.changeDurationBySeconds(this.defaultSkipInterval * this.player.playbackRate() * 2)
             break
+          case '<':
           case 'O':
           case 'o':
             // Decrease playback rate by 0.25x
             event.preventDefault()
             this.changePlayBackRate(-this.videoPlaybackRateInterval)
             break
-          case 'P':
```

Comment:
Should probably readd this and do a check for `event.shiftKey` in the case statement so if capslock is on it will still work